src/main/java/io/github/w7mike/model/JobGroups.java

```diff
@@ -26,10 +26,17 @@ public JobGroups() {
         return jobs;
     }
 
-     public void setJobs(final Set<Job> jobSet) {
+    public void setJobs(final Set<Job> jobSet) {
         this.jobs = jobSet;
     }
 
+    public Projects getProjects() {
+        return projects;
+    }
+
+    public void setProjects(final Projects projects) {
```

Comment:
There is nothing wrong with making method's parameters `final` but in case of simple setters, there is also no gain.